### DISCUSSION TITLE
✅ custom templates | dynamic data

### DISCUSSION THREAD

**Discussion by webmasterbart** [elementor/elementor#14046] [Feature Request]
WordPress/Gutenberg is catching up on things here.
DYNAMIC DATA,  custom content templates, ...
https://generatepress.com/introducing-the-gp-theme-builder/

So I ask myself over and over again : 
why are functionalities like ELE Custom Skin not in the core of Elementor yet ?
https://wordpress.org/plugins/ele-custom-skin/

To keep ahead of Gutenberg, the Elementor dev needs to keep pushing !

**Comment by nicholaszein** [comment]
Hello,

the Loop Builder feature was added to Elementor Pro in version 3.8.0.

Feel free to activate it on your installation.
